src/System.Text.Encodings.Web.Utf8/UrlDecoder.cs

```diff
@@ -2,7 +2,11 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
-using System.Text.Encodings.Web.Buffers;
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
```

Comment:
remove it please

Reply:
This is a standard license we use in corfxlab, isn't it?

Reply:
Its duplicated.